lib/certUtils.js

```diff
@@ -79,11 +78,19 @@ class Certificate {
 	}
 
 	getIssuer() {
-		return this._cert.issuer.typesAndValues[0].value.valueBlock.value;
+		return this.searchForCommonName(this._cert.issuer.typesAndValues);
 	}
 
-	getSerial() {
-		return this._cert.subject.typesAndValues[0].value.valueBlock.value;
+	getSerial(compatability) {
+		if (compatability === undefined) {
+			console.warn("[DEPRECATION WARNING] Please use getSerial(\"v1\") or getSerial(\"v2\").");
```

Comment:
I think it might reduce confusion and trial & error effort to tell people to go straight to use v2

Reply:
Done.